microcosm_flask/swagger/schemas.py

```diff
@@ -62,7 +62,8 @@ def iter_fields(self, schema: Schema) -> Iterable[Tuple[str, Field]]:
         """
         for name in sorted(schema.fields.keys()):
             field = schema.fields[name]
-            yield field.dump_to or name, field
+            # XXX: dump_to was removed; it wasn't being used anywhere? double-check
```

Comment:
Pay attention to me.

Reply:
I think you're safe: https://github.com/search?q=org%3Aglobality-corp+dump_to&type=Code I only see it used in this repo, on this line.